From 471ed9a12c2777f1ed7c7c51a2a63e754b9e00e7 Mon Sep 17 00:00:00 2001
From: Matthieu SCARSET <m@matthieuscarset.com>
Date: Thu, 19 Oct 2017 22:10:43 -0400
Subject: [PATCH 1/1] Remove translations install task

---
 config_installer.profile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config_installer.profile b/config_installer.profile
index 610282b..9dc812a 100644
--- a/config_installer.profile
+++ b/config_installer.profile
@@ -23,6 +23,8 @@ include_once __DIR__ . '/src/Storage/SourceStorage.php';
  * Implements hook_install_tasks_alter().
  */
 function config_installer_install_tasks_alter(&$tasks, $install_state) {
+  unset($tasks['install_import_translations']);
+  unset($tasks['install_finish_translations']);  
   unset($tasks['install_download_translation']);
   $key = array_search('install_profile_modules', array_keys($tasks), TRUE);
   unset($tasks['install_profile_modules']);
-- 
2.11.0

